packages/lesswrong/components/posts/CategoryDisplay.jsx

```diff
@@ -29,6 +29,21 @@ const styles = theme => ({
   unread: {
     color: theme.palette.secondary.light,
   },
+  container: {
+    width: 50,
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    cursor: "pointer",
+  
+    "&:hover": {
+      backgroundColor: "rgba(0,0,0,.05)",
+    }
+  },
+  categoryTooltip: {
+    lineHeight: 14,
```

Comment:
This will cause the line-height to be `14 * em` which will make it really really large.